packages/neuron-wallet/src/services/cells.ts

```diff
@@ -295,7 +295,10 @@ export default class CellsService {
       input: Input,
       witness: WitnessArgs,
     },
-    lockCodeHashes: string[] = [SystemScriptInfo.SECP_CODE_HASH]
+    lockClass: {
```

Comment:
What's the meaning of `lockClass`?

Reply:
It means the combination of code hash and hash type under a lock. This is meant to filter the cells with these two data points. I tried to avoid calling the `lockType`, the wording of which would be too confusing with the other concepts on CKB.